README.md

```diff
@@ -16,41 +14,92 @@ pip install torchvision
+* Some additional [metadata files](https://surfdrive.surf.nl/files/index.php/s/EF1bA9YYfhiBxoN).
 
-### Get data
-Download the Flickr8k data.
-For testing, we prepared the [Flickr1d subset](https://github.com/spokenlanguage/flickr1d) of 10 Flickr8k images and corresponding captions.
+Create a folder to store the dataset (we will assume here that the folder is
+`~/corpora/flickr8k`)  and move all the files you downloaded there, then
+extract the content of the archives.  You can now setup the environment and
+start preprocessing the data.
 
 ### Configuration
-We use ConfigArgParse for setting necessary input variables.
-This means you can use either a configuration file (config.ini or config.yml), environment variables or command line arguments to specify the necessary configuration parameters.
-For instance, to use environment variables with the Flickr1d dataset, set the PLATALEA_DATA_ROOT variable to the directory in which the Flickr1d data was downloaded, e.g. if it was cloned at `/user/data/flickr1d` do:
+
+We use ConfigArgParse for setting necessary input variables, including the
+location of the dataset.  This means you can use either a configuration file
+(config.ini or config.yml), environment variables or command line arguments to
+specify the necessary configuration parameters.
+
+To specify the location of the dataset, one option is to create a configuration
+file under your home directory (`~/.config/platalea/config.yml`)., with
+follwing content:
+
+```
+flickr8k_root   /home/<user>/corpora/flickr8k
+```
+
+The same result can be achieved with an environment variable:
 
 ```sh
-export FLICKR8K_ROOT=/user/data/flickr1d
-export FLICKR8K_METADATA_JSON=dataset.json
-export PLATALEA_AUDIO_FEATURES_FN=mfcc_features.pt
+export FLICKR8K_ROOT=/home/<user>/corpora/flickr8k
 ```
 
-You could also specify these options on the command line directly when running (the respective options would be `--flickr8k_root=...`, `--flickr8k_meta=...` and `--audio_features_fn=...`) or via a configuration file (using the names of the command line options without leading `--`).
+You could also specify this option directly on the command line when running
+an experiment (the respective options would be `--flickr8k_root=...`).
 
-### Experiment dependencies
-Different experiments may have different additional dependencies.
-The `basic-stack` experiment needs the following:
+### Preprocessing
 
-```sh
-pip install sklearn python-Levenshtein
+Run the preprocessing script to extract input features:
+
+```bash
+python platalea/utils/preprocessing.py flickr8k
 ```
 
-### Run
-Then finally, run the experiment:
+## Training
+
+You can now train a model using one of the examples provided under
+`platalea/experiments`, e.g.:
 
 ```sh
-python experiments/basic-stack/run.py
+cd platalea/experiments/flickr8k
+mkdir -p runs/test
+cp basic.py runs/test
+cd runs/test
+python basic.py
```

Comment:
~I think it should be `python basic_default.py`. I'm all in favor of renaming that to `basic.py` btw :)~ I see you renamed it, good!

Apart from this, though, there is a more important comment to make here: while it is still possible to run the scripts from the source directory, when you have installed the source, you don't actually need to know the source directory anymore. You can just run with `python -m platalea.experiments.flickr8k.basic` from any directory you like. That also saves you the trouble of having to make a copy.

Reply:
Updated